labs/iot/gps-lab/client/code/raspberry-pi-gps-tracker.py

```diff
@@ -0,0 +1,95 @@
+import time
+import serial
+
+# Using the Python Device SDK for IoT Hub:
+#   https://github.com/Azure/azure-iot-sdk-python
+# The sample connects to a device-specific MQTT endpoint on your IoT Hub.
+from azure.iot.device import IoTHubDeviceClient, Message
+
+# The device connection string to authenticate the device with your IoT hub.
+CONNECTION_STRING = "<Device_Connection_String>"
```

Comment:
Can this me moved to a .env file to be more best practice aligned? Then use python-dotenv to load it?